src/main/java/com/textmessenger/config/WebConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package com.textmessenger.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebConfiguration implements WebMvcConfigurer {
+
+  @Override
+  public void addViewControllers(ViewControllerRegistry registry) {
+    registry.addViewController("/{spring:\\w+}")
+            .setViewName("forward:/");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal "forward:/" 3 times. ([squid:S1192](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=226&issues=AWX8wPRw_XXc3YMrkI9N&open=AWX8wPRw_XXc3YMrkI9N)

[ISSUE_KEY]: AWX8wPRw_XXc3YMrkI9N